File: Practices/time_of_day.py
#CC 6th Time of Day

time = int(input("What is the time now in the 24 hour format?"))

if time >=0 and time<12:
    print("Good Morning!")
elif time <=17 and time>=12:
    print("Good Afternoon!")
elif time >17 and time<24:
    print("Good Evening!")
else:
    print("You got school in the morning, GO TO BEDD!!")

File: c_programming/c_programming/practices/time_of_day.c
//CC 6th time of day practice
#include <stdio.h>
#include <string.h>

int main(void){
    int time;
    char name[1000];
    printf("What is the time in military form without colons??: ");
    scanf("%d", &time);

        if (time >= 0000) {
        printf("Good morningg!\n");
     } else if (time >= 1200) {
        printf("Good afternoon!\n");
    } else if (time >= 1800 ) {
        printf("Good evening!\n");  
    } else if (time >= 2100 ) {
        printf("Good night!\n");
    } else {
        printf("Please enter a real time.\n");
    }


    return 0; 
}
